test/test.dart

```diff
@@ -579,26 +578,33 @@ Future<void> baasTest(
   } else if (skip is bool) {
     if (url == null) skip = "BAAS URL not present";
   }
+  final baasUri = Uri.parse(uriVariable!);
 
   test(name, () async {
-    final config = await getAppConfig(appName: appName);
-    await testFunction(config);
+    final app = baasApps[appName.name] ??
+        baasApps.values.firstWhere((element) => element.name == BaasClient.defaultAppName, orElse: () => throw RealmError("No BAAS apps"));
+    try {
+      if (app.error != null) {
+        throw app.error!;
+      }
+      final config = await getAppConfig(baasUri, app.clientAppId);
```

Comment:
This is introducing too much work on the caller. Why did we change the signature of `getAppConfig`. It will be much nicer if we keep the getAppConfig by name and do this filtering inside it.

Reply:
Done